pkg/kapp/resources/identified_resources_list.go

```diff
@@ -55,6 +55,16 @@ func (r IdentifiedResources) List(labelSelector labels.Selector, resRefs []Resou
 		}
 	}
 
+	// Dump any resources that do not contain Labels
+	// as they cannot be kapp resources
+	var newResources []Resource
+	for _, res := range resources {
+		if len(res.Labels()) > 0 {
```

Comment:
hmm, this would break usage of `kapp -a label:` or other negative-matching variants.